main/src/main/java/cgeo/geocaching/EditWaypointActivity.java

```diff
@@ -391,11 +396,28 @@ public static ImmutableTriple<FormulaEditText, FormulaEditText, TextSpinner<Dist
         }
     }
 
+    private Set<String> getNeededVariablesForProjection() {
+        final Set<String> projectionVars = new HashSet<>();
+
+        final ImmutableTriple<FormulaEditText, FormulaEditText, TextSpinner<DistanceUnit>> fields = getFieldsForProjectionType();
+        if (fields.left == null) {
```

Comment:
I assume the check has to be then other way around? (`... !=null`)
As is the code will produce an NPE in next line if `fields.left` is null

Reply:
Ups

Reply:
And the vars will not be added :-o